speedline.d.ts

```diff
@@ -0,0 +1,37 @@
+
+interface SpeedlineOutput {
+  beginning: number;
+  end: number;
+  speedIndex: number;
```

Comment:
this will have to be `speedIndex?: number;` and need a `perceptualSpeedIndex?: number;` additionally. There might be a clever way to do it based on the options, but that's correct, at least :)